src/pre_commit_terraform/terraform_docs_replace.py

```
@@ -1,76 +1,97 @@
     for filename in cast_to(list[str], parsed_cli_args.filenames):
-        if (os.path.realpath(filename) not in dirs and
-                (filename.endswith(".tf") or filename.endswith(".tfvars"))):
+        if os.path.realpath(filename) not in dirs and (
+            filename.endswith('.tf') or filename.endswith('.tfvars')
+        ):
             dirs.append(os.path.dirname(filename))
 
     retval = ReturnCode.OK
 
-    for dir in dirs:
+    for directory in dirs:
```

Ruff (0.8.2)

Loop control variable `directory` not used within loop body

Rename unused `directory` to `_directory`

(B007)